flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -143,9 +148,83 @@ def _create_volume(self, client, name, driver_opts):
             'DriverOpts': driver_opts,
         }
         result = client._result(client._post_json(url, data=data), True)
-        self.addCleanup(client.remove_volume, name)
+        if cleanup:
+            self.addCleanup(client.remove_volume, name)
         return result
 
+    def _test_sized_vol_container(self, cluster, node):
+        """
+        Create a volume with a size, then create a
+        container that can use that volume and run lsbk to test
+        its size. An http server inside the container uses lsblk to
+        return the size of the volume when path is not "/is_http"
+
+        :param cluster: flocker cluster
+        :param node: node the contianer and volume will be on.
+        """
+        client = get_docker_client(cluster, node.public_address)
+        volume_name = random_name(self)
+        # Because flocker/control/schema/types.yml#L250 > 64MiB
+        # "k" and "t" are too big and too small for test
+        # but we to keep some randomness to this test.
+        size_expressions = list(chain.from_iterable((x, x+"b", x+"ib")
+                                for x in ["g"]))
```

Comment:
@tomprince im not familiar with `strategy.example()`. 

On the other hand, if we keep the strategy sample small enough maybe we could run this multiple times, but then again maybe not. Open to other ways.